src/BasicRum/TechnicalMetrics/TimeToFirstPaint.php

```diff
@@ -15,4 +15,9 @@ public function getSelectTableName(): string
     {
         return 'navigation_timings';
     }
+
+    public function getSchemaEntry(): string
```

Comment:
What is the purpose of this function? This type of classes should not contain things that are related to the JSON schema but the name of the functions is containing "Schema".

Reply:
It was added by mistake